app/src/main/java/com/github/se/icebreakrr/MainActivity.kt

```diff
@@ -246,6 +244,12 @@ fun IcebreakrrNavHost(
         route = Route.AROUND_YOU,
     ) {
       composable(Screen.AROUND_YOU) {
+        profileViewModel.getSelfProfile()
+        profileViewModel.getInboxOfSelfProfile()
+        meetingRequestViewModel?.setInitialValues(
+            MeetingRequestManager.ourFcmToken ?: "null",
+            MeetingRequestManager.ourUid ?: "null",
+            MeetingRequestManager.ourName ?: "null")
```

Comment:
Ok this has been updated at the time i'm writting my comment but the idea is that we need to get the token, uid and name from the SignIn screen (the Manager does that already) and then we need to set the values in the Meeting Request VM as soon as possible. That is why I set those values up in the AroundYou, because it has to be done after the Auth/Sign In step